### ISSUE TITLE
Premade raspberry pi image for rapid field deployment

### ISSUE THREAD

**Issue by mofosyne** [arboliva/AirChat#2]
Something to consider. Rolling a raspberry pi image for the raspberry pi, will help in deploying this technology in the field. This is since, all the user has to do, is flash the image to an SD card, and place it into the raspberry pi. (The sd card holds the linux image, and is equivalent to a HDD in a laptop)

Another thing to consider in terms of deployment is that the raspberry pi has a consistent form factor, and large passionate community supporting it. If you haven't heard of it, it's a dirt cheap ARM linux board designed to help kids in developing countries learn how to program. (Has sound out, HDMI, usb, ethernet, etc... but no sound input. But you can insert a usb microphone/soundcard for that)

Btw there is also a compute module that the raspberry pi foundation is releasing, which would be even more energy efficient, and allow for really compact designs. (Only issue is you need to construct your own PCB, as this is more for industrial applications)

This is best suited for relays and 'field' BBS or community kiosk. The raspberry pi is energy efficient and small, allowing for more space for a battery and solar panels in a suitcase (Potentially allowing for field deployable stations with multi day runtime capability). 

> > tl;dr: Rapid deployment capability is paramount in time critical events, especially for non technical users. Raspberry Pi + usb soundcard does the above, by making software deployment and maintenance a single SD card affair. Simply put, the user cannot brick the raspberry pi.

**Comment by arboliva** [comment]
we tested it on raspberry pi with external usb soundcard already and it works great!
we would prefer some external and trusted third party publish a raspberry pi project based on airchat to avoid the usual mistrusting issues. 
we are looking forward for it.
